### ISSUE TITLE
More intelligently suppress messaging during runs where no existing data is found

### ISSUE THREAD

**Issue by stkent** [stkent/GooglePlayReviewScraper#2]
At the moment, the script reports all reviews as 'new' and the latest app version as 'newly released'. We should probably suppress this messaging and send something more useful, like "Tracking initialized for app <APP_NAME>".

**Comment by macklinu** [comment]
This would mean we would need to store our fetched data in some sort of datastore to compare previous days. We could use something like Redis or Postgres. Do you have thoughts on a particular style of implementation?

**Comment by stkent** [comment]
Right now, the app writes json to a local file, so either of those options should be ok. Are we looking for this app to store any historical information?

If not, we only need to store the most recently inspected ratings.

If so, we'll need to index by date, say, as in the current implementation.

**Comment by macklinu** [comment]
If we only store the previous day's worth of ratings, then a simple file should be fine. I forgot to look at the `diskops.py` file before I commented on this issue and started to overcomplicate it haha. :stuck_out_tongue:

**Comment by stkent** [comment]
If we write to a file, does that persist across Heroku's executions of the script? I have not set up anything on Heroku before, so I'm not sure what we have available in terms of persistence.

**Comment by macklinu** [comment]
This is a good question to research. I'll look into this.

**Comment by stkent** [comment]
:+1:

**Comment by macklinu** [comment]
Yeah, I don't think storing files like this would be a smart idea on Heroku. I don't think we're guaranteed the file will exist across spinning up different instances. We would probably be better off with a simple key-value datastore like [Redis](https://devcenter.heroku.com/articles/redistogo#install-redis-in-python).

**Comment by stkent** [comment]
Let's go with redis. I'll make a separate issue for that. This one is really to prevent the script from reporting 10000s of new reviews when it is first run :smile_cat:

**Comment by stkent** [comment]
Done, see #15
